Google.PowerShell.IntegrationTests/Logging/Logging.GcLog.Tests.ps1

```diff
@@ -0,0 +1,102 @@
+﻿. $PSScriptRoot\..\GcloudCmdlets.ps1
+Install-GCloudCmdlets
+
+$project, $zone, $oldActiveConfig, $configName = Set-GCloudConfig
+
+Describe "Get-GcLogEntry" {
+    $r = Get-Random
+    $script:logName = "gcp-testing-log-$r"
+    $script:secondLogName = "gcp-testing-log2-$r"
+    $textPayload = "Test entry"
+    $secondTextPayload = "Second test entry"
+    $jsonPayload = "{\`"Key\`":\`"Value\`"}"
+    $timeBeforeCreatingLogs = [DateTime]::Now
+    gcloud beta logging write $logName $textPayload --severity=ALERT > $null
+    gcloud beta logging write $logName $jsonPayload --severity=INFO --payload-type=json
+    gcloud beta logging write $secondLogName $secondTextPayload --severity=ALERT
+    gcloud beta logging write $secondLogName $jsonPayload --severity=INFO --payload-type=json
+    # We add 2 minutes to account for delay in log creation
+    $timeAfterCreatingLogs = [DateTime]::Now.AddMinutes(2)
```

Comment:
This is only for the test that checks for log entries created after a certain period of time so I don't think that is necessary.